.yarnrc.yml

```diff
@@ -0,0 +1,25 @@
+cacheFolder: ~/.yarn/berry/cache
+
+compressionLevel: 0
+
+enableGlobalCache: true
```

Comment:
Do we need to set global folder here if this flag is true : 
https://yarnpkg.com/configuration/yarnrc#global